src/open_inwoner/openklant/models.py

```diff
@@ -213,6 +221,11 @@ class OpenKlant2Config(SingletonModel):
         ),
     )
 
+    register_api_required_fields = ("service",)
+
+    def has_api_configuration(self):
+        return all(getattr(self, f, "") for f in self.register_api_required_fields)
```

Comment:
nitpick, but we can make the intent clearer:
```suggestion
        return all(getattr(self, f, False) for f in self.register_api_required_fields)
```